introduction_to_applying_machine_learning/xgboost_customer_churn/xgboost_customer_churn.ipynb

```diff
@@ -49,7 +49,6 @@
    "cell_type": "code",
```

Comment:
<p>why neo support code is removed? is Neo available in all AWS regions now?</p><p> </p><p>also, please change the <code>get_image_uri</code>  to the latest SDK v2 contract.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>